src/daemon-command/kill-process.ts

```diff
@@ -0,0 +1,17 @@
+import { Socket } from "net";
+
+import { Daemon } from "../commands/start-daemon.command";
+
+export async function killProcess(daemon: Daemon, socket: Socket | undefined, scriptName: string): Promise<void> {
+    const childProcess = daemon.processes[scriptName];
+    if (childProcess && !childProcess.killed) {
+        console.log(`killing ${scriptName}`);
```

Comment:
Please use log style used throughout the project

Reply:
Actually I'm not a fan of those colors, especially when used blindly everywhere.

It can be useful tough if you have a mixture of the output of different child processes + the parent process - so you can identify easily which process the output is coming from.

And for debug output (that happens within the daemon) - that a end use will usually not see - it's questionable if it's worth the effort.

Reply:
I would use special formatting / colors for system outputs so it's easily identifiable in longer logs

Reply:
yes, that is useful!

But only if there actually are longer logs from different sources.

I will come back to that in a later PR.